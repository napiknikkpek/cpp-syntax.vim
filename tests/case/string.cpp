auto c = 'a';
auto script = "as\"df";
auto script = u8"asdf";
auto script = U"asdf";
auto script = R"delimiter(asdf
asdf
asdf
)delimiter";
