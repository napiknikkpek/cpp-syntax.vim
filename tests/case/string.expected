aaaa.x...ccc.
aaaa.xxxxxx...zssssssz.
aaaa.xxxxxx...zzzssssz.
aaaa.xxxxxx...zzssssz.
aaaa.xxxxxx...zzzzzzzzzzzzssss
ssss
ssss
zzzzzzzzzzz.
